.maintain/frame-weight-template.hbs

```diff
@@ -0,0 +1,114 @@
+// This file is part of Substrate.
```

Comment:
how is this used?

Reply:
@sinkingsugar This is weight template file and used in below command to generate weights. 

`./target/release/clamor benchmark --chain=dev --execution=wasm --wasm-execution=compiled --pallet=fragments-pallet --extrinsic='*' --steps=50 --repeat=20 --heap-pages=4096 --template=./.maintain/frame-weight-template.hbs --output=./pallets/fragments/src/weights.rs`